src/ontology/cl-edit.owl

```diff
@@ -32019,6 +32024,56 @@ AnnotationAssertion(rdfs:label obo:CL_4042006 "dural macrophage"@en)
 SubClassOf(obo:CL_4042006 obo:CL_4042003)
 SubClassOf(obo:CL_4042006 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002092))
 
+# Class: obo:CL_4042007 (protoplasmic astrocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:25904839") Annotation(oboInOwl:hasDbXref "PMID:34616062") Annotation(oboInOwl:hasDbXref "PMID:37139179") Annotation(oboInOwl:hasDbXref "PMID:37824655") obo:IAO_0000115 obo:CL_4042007 "An astrocyte cell that has highly branched protrusions, its soma is part of a neocortex and it is found between layer 2 and 6. A protoplasmic astrocyte is involved in the formation and elimination of synapses, in the clearance of glutamate, in the modulation of synaptic functions and in the regulation of blood flow in response to synaptic activity.")
```

Comment:
Suggest: " A cell type with highly branched protrusions, found in neocortex layers 2-6, typically involved with the formation and elimination of synapses, glutamate clearance, modulation of synaptic functions and regulation of blood flow in response to synaptic activity."